paths2openscad.py

```diff
@@ -997,9 +1049,67 @@ def effect(self):
 
             # The module that calls all the other ones.
             self.f.write('}\n\n%s(%s);\n' % (name, self.options.height))
+            self.f.close()
 
         except:
-            inkex.errormsg('Unable to open the file ' + self.options.fname)
+            inkex.errormsg('Unable to write file ' + self.options.fname)
+
+        if self.options.scad2stl == 'true' or self.options.stlpost == 'true':
+            stl_fname = re.sub(r"\.SCAD", "", full_fname, flags=re.I) + '.stl'
+            cmd = self.options.scad2stlcmd.format(
+                **{'SCAD': full_fname, 'STL': stl_fname})
```

Comment:
With python 2.7 and also 3.5 it looks like this for me:
<pre>
Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> "foo {SCAD} bar".format('SCAD': 'here')
  File "<stdin>", line 1
    "foo {SCAD} bar".format('SCAD': 'here')
                                  ^
SyntaxError: invalid syntax
>>> "foo {SCAD} bar".format(**{'SCAD': 'here'})
'foo here bar'
>>> 
</pre>

Is there some import that makes the use for .format nicer?